packages/node-integration-tests/suites/express/sentry-trace/baggage-header-out/server.ts

```diff
@@ -27,6 +27,7 @@ app.get('/test/express', (_req, res) => {
   const transaction = Sentry.getCurrentHub().getScope()?.getTransaction();
   if (transaction) {
     transaction.traceId = '86f39e84263a4de99c326acab3bfe3bd';
+    transaction.metadata.source = undefined;
```

Comment:
Why do we need to add this?

Reply:
Because [this test](https://github.com/getsentry/sentry-javascript/blob/0f4d234342f8a3573ca9a8e39be6e31bdc091b43/packages/node-integration-tests/suites/express/sentry-trace/baggage-header-out/test.ts#L22) was ensuring that transaction names without a valid source are not added to the DSC.

But I'll split the two tests for which this server file is made up so that we only set this for one of them.